app/views/hyrax/my/collections/index.html.erb

```diff
@@ -41,7 +41,13 @@
   <%# Collections list %>
   <div class="panel panel-default collections-panel-wrapper">
     <div class="panel-heading">
-      <h3 class="panel-title"><strong><%= @response.total_count %> collections</strong> in the repository</h3>
+      <% if  current_page?(hyrax.my_collections_path(locale: nil)) %>
+         <h3 class="panel-title"><strong><%= @response.total_count %> collections</strong> you own in the repository</h3>
+      <% elsif current_page?(hyrax.dashboard_collections_path(locale: nil)) && current_ability.admin? %>
+          <h3 class="panel-title"><strong><%= @response.total_count %> collections</strong> in the repository</h3>
+      <% else %>
+          <h3 class="panel-title"><strong><%= @response.total_count %> collections</strong> you can manage in the repository</h3>     
+      <% end %>
```

Comment:
Can you update these messages to come from translations?

Reply:
I wonder if the conditionals could be reordered to have the final `else` condition produce text `collections in the repository`.  Seems like the most general statement and if something slipped through the conditionals, it would be better to land on this statement than the `you can manage` statement.